features/user_login.feature

```diff
@@ -10,7 +10,7 @@ Feature: Login
 
   Scenario: Access the login page
     Given I am on the home page
-    And I click "Login"
+    And I click on "Login"
```

Comment:
@diraulo to make it match with a step definition that was already created in another feature